dbms/src/Functions/h3EdgeArc.cpp

```diff
@@ -0,0 +1,80 @@
+#include "config_functions.h"
+#if USE_H3
+#include <Columns/ColumnsNumber.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/IFunction.h>
+#include <Common/typeid_cast.h>
+#include <ext/range.h>
+
+
+extern "C"
+{
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
```

Comment:
Maybe mark the include directory as SYSTEM in CMakeLists will help.

Reply:
I just copy-paste this part from `geoToH3.cpp`, can you explain how should I fix it?